database/sqlite/sqlite_aclk_alert.c

```diff
@@ -239,6 +243,9 @@ void aclk_push_alert_event(struct aclk_database_worker_config *wc, struct aclk_d
 #else
     int rc;
 
+    if (wc->pause_alert_updates)
```

Comment:
This will still generate the opcode to check for alerts

Reply:
Hi Stelio, ok, moved the check before creating the opcode.